core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/KotlinNotebookPluginUtils.kt

```diff
@@ -60,7 +59,7 @@ public object KotlinNotebookPluginUtils {
      * It's used for example for dynamic pagination in Kotlin Notebook Plugin.
      */
     public fun getRowsSubsetForRendering(df: AnyFrame, startIdx: Int, endIdx: Int): DisableRowsLimitWrapper =
-        DisableRowsLimitWrapper(df.filter { it.index() in startIdx until endIdx })
+        DisableRowsLimitWrapper(df[startIdx until endIdx])
```

Comment:
cool place to try the new `..<` operator :D

Reply:
new shiny! i'll use it indeed